src/components/Paymethods.tsx

```diff
@@ -0,0 +1,57 @@
+import { number } from "io-ts";
+import { Integer } from "italia-ts-commons/lib/numbers";
+import React, { useState } from "react";
+import { TabContent, TabPane, Nav, NavItem, NavLink } from "reactstrap";
+import PaymentMethodTabPane from "./PaymentMethodTabPane";
+import { format, parseISO } from "date-fns";
+import { useTranslation } from "react-i18next";
+import "./Paymethods.css";
+import { PaymentMethod } from "../generated/definitions/PaymentMethod";
+import classNames from "classnames";
+
+function Paymethods(props) {
+  const [activeTab, setActiveTab] = useState<number>(0);
+
+  const { t, i18n } = useTranslation();
+
+  const toggle = (tab: number) => {
+    if (activeTab !== tab) setActiveTab(tab);
+  };
+
+  return (
+    <>
+      <div className="mt-4 mb-2 font-weight-bold">{t("Payment methods")}</div>
+
+      <Nav tabs className="PaymentMethod__list">
+        {props.paylist.map((el: PaymentMethod, index: number) => (
+          <NavItem key={index}>
+            <NavLink
+              className={classNames({
+                "d-flex align-items-center position-relative": true,
+                active: activeTab === index
+              })}
+              title="Visualizza dettaglio"
```

Comment:
```suggestion
              title="{t("Details")}"
```

not required with this PR, but a better approach with locales is to have a dedicated module with some hierarchy (that's the approach we follow with IO app), see https://github.com/AgID/agid-forms-frontend/blob/master/src/locales/it.ts